server/app.js

```diff
@@ -37,4 +39,176 @@ io.on('connection', (socket) => {
             socket.emit("alert text", "Authentication failed. Please try again.");
         }
     });
-});
\ No newline at end of file
+
+    // start of added code to talk with poker.vue
+    // these next two functions are temporary, need to login and join table via login page and tables page
+    socket.on("logIn", function(msg){
+        login(socket, msg);
+    });
+    socket.on('joinTable', function(msg){ // a user wishes to join a table
+        joinTable(io, socket, msg);
+    });
+
+    // each player will send a message upon a game play button click, this will check whether or not it is the correct player's turn and if the move is valid
+    socket.on('turnDecision', function(msg){
+        turnDecision(io, socket, msg);
+    });
+
+});
+
+
+// a list of sample profiles already put into the system, these are not the real profiles (just hard coded ones)
+// need to update to use the real profiles stored in the server
+let profiles = [
+    {userID: 1000, name: 'Mark123', chips: 20000, icon: 'player_icon_1'},
+    {userID: 2000, name: 'John456', chips: 40000, icon: 'player_icon_1'},
+    {userID: 3000, name: 'Luke854', chips: 60000, icon: 'player_icon_1'},
+    {userID: 4000, name: 'Jane027', chips: 80000, icon: 'player_icon_1'},
+    {userID: 5000, name: 'Stan693', chips: 10000, icon: 'player_icon_1'},
+    {userID: 6000, name: 'Mack273', chips: 20, icon: 'player_icon_1'},
+];
+// the entire list of tables, might need a better way to store them?
+let tables = [];
+tables.push(new PokerTable()); // initially we will have one table, when socket rooms are implemented then multiple tables will be added
+
+
+// these next three functions are for signing up a client to the poker table, need to use the real login and tables page to do this
+function login(socket, msg){
+    let profile = profiles.find(profile => profile.userID === msg);
+    if(profile === undefined){// the user does not exist
+        return;
+    }
+    else{// send the current table state
+        console.log('emitted');
+        socket.emit('tableState', JSON.stringify(tables[0].getTableState()));
+    }
+}
```

Comment:
I believe login is in master - you'll likely have merge conflicts.

Reply:
i don't see it in master

Reply:
have you pulled?

Reply:
I've pulled master and didn't have conflicts